src/Symfony/Component/HttpFoundation/Session/Flash/AutoExpireFlashBag.php

```diff
@@ -78,7 +78,7 @@ public function peek($type, array $default = [])
      */
     public function peekAll()
     {
-        return \array_key_exists('display', $this->flashes) ? (array) $this->flashes['display'] : [];
+        return \array_key_exists('display', $this->flashes) ? $this->flashes['display'] : [];
```

Comment:
The cast is here since the first implementation but it's not casted everywhere. I mean `->flashes['display']` is accessed as an array everywhere else without any check or cast. So it's useless to cast it here, it's always an array.